### PR TITLE
Strings: “Completed” to “Uploaded” / “Downlaoded”

### PR DESCRIPTION
**Pull request by da2x**

### CODE REVIEW

**Comment by iNPUTmice** [comment]
This PR also changes the string ```title_activity_contact_details``` on accident.

**Comment by d9h02f** [comment]
At least to me, "70% completed" sounds a lot better than "70% uploaded" or "70% downloaded". Why this change?

**Comment by stevenroose** [comment]
The code also not correctly reflects the title.

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 	<string name="title_activity_manage_accounts">Manage Accounts</string>
 	<string name="title_activity_settings">Settings</string>
 	<string name="title_activity_conference_details">Conference Details</string>
-	<string name="title_activity_contact_details">Contact Details</string>
+	<string name="title_activity_contact_details">Contacct Details</string>
```
Comment (benediktg) [inline src/main/res/values/strings.xml]: I think here is a typo…

**Pull request closed without merging**